### PR TITLE
Rename "development environment" to "development container"

### PR DESCRIPTION
**Pull request by pchico83**
Signed-off-by: Pablo Chico de Guzman <pchico83@gmail.com>

## Proposed changes
- Rename "development environment" to "development container".  The development environment is a wider concept.

### CODE REVIEW

File: cmd/restart.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func Restart() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "restart",
-		Short: "Restarts the pods of your development environment",
+		Short: "Restarts the services of your okteto manifest",
```
Comment (rlamana) [inline cmd/restart.go:34]: This is the only one that sounds odd. The rest of the changes are much more clear now.
What about `Restarts the pods containing your development container`? or `Restart the pod of your development container` or something like that... I'm not sure if my proposal is accurate enough.

Comment (pchico83) [reply cmd/restart.go:34]: This command restarts the pods indicated by the `services` field of the okteto manifest. It does not restart the dev container. What about something like:
`Restarts the pods references by the "services" field of the okteto manifest`

Comment (rlamana) [reply cmd/restart.go:34]: Sounds more accurate like that then

Comment (rberrelleza) [reply cmd/restart.go:34]: In kubernetes you don't restart services, you recreate (or restart) the pods. I think we should use more  kubernetes-specific in this kind of descriptions.

Comment (rberrelleza) [reply cmd/restart.go:34]: Maybe: restarts the deployment listed in the `services` field of the okteto manifest?

**Pull request merged**